openapi/api.yaml

```diff
@@ -325,6 +345,88 @@ components:
       in: header
       name: X-Xsrf-Token
   schemas:
+    Process:
+      type: object
+      properties:
+        name:
+          type: string
+        description:
+          type: string
+        applicability:
+          type: string
+        stages:
+          type: array
+          items:
+            $ref: '#/components/schemas/ProcessStage'
+    ProcessStage:
+      type: object
+      properties:
+        name:
+          type: string
+        description:
+          type: string
+        progress_items:
+          type: array
+          items:
+            $ref: '#/components/schemas/ProgressItem'
+        actions:
+          type: array
+          items:
+            $ref: '#/components/schemas/Action'
+        approvals:
+          type: array
+          items:
+            $ref: '#/components/schemas/GateInfo'
+        incoming_stage:
+          type: integer
+        outgoing_stage:
+          type: integer
+        stage_type:
+          type: integer
+          nullable: true
+    ProgressItem:
+      type: object
+      properties:
+        name:
+          type: string
+        field:
+          type: string
+    Action:
+      type: object
+      properties:
+        name:
+          type: string
+        url:
+          type: string
+        prerequisites:
+          type: array
+          items:
+            type: string
+    GateInfo:
+      type: object
+      properties:
+        name:
+          type: string
+        description:
+          type: string
+        gate_type:
+          type: integer
+        rule:
+          type: string
+        approvers:
+          type: array
+          items:
+            type: string
```

Comment:
This field failed when I ran the tests myself, and the tests were actually looking for a string type, rather than a list of strings

Reply:
I set it to array because the in approval_defs.py it's defined as either string / list[str]. It seems like it won't be a list given the tests we had. I typed them as string and it worked.

Reply:
This may be an issue we need to look at before landing. Are there different circumstances where a string is passed over a list of strings? And do we need to facilitate both?

Reply:
yes in [approval_def.py](https://github.com/GoogleChrome/chromium-dashboard/blob/dd4a0a4d6b0fe292b51411bb1d84a36da0205857/internals/approval_defs.py#L114), approvers could either be a single email address or a list of such, but it seems like test cases are only looking for a single string